core/src/calleeFunctions/helpers/uniswapAutoRouter.ts

```diff
@@ -91,6 +91,11 @@ const _fetchAutoRouteInformation = async function (
             throw new Error('Only V3 routes are supported.');
         }
         const fees = bestRoute.route.pools.map(pool => pool.fee);
+        const pools = bestRoute.route.pools.map(pool => ({
+            addresses: [pool.token1.address, pool.token0.address],
+            fee: pool.fee,
+            routes: [pool.token1.symbol, pool.token0.symbol],
+        }));
```

Comment:
this intends to address https://github.com/sidestream-tech/unified-auctions-ui/pull/531#discussion_r1039876930

Reply:
Thanks!